File: max/max.js
'use strict';

module.exports = function(array) {
  var maximum = array[0];

  for (var i = 1, l = array.length; i < l; i++) {
    var el = array[i];
    if (maximum < el) {
      maximum = el;
    }
  }

  return maximum;
};

File: max/max.c
int max(int array[], int length) {
  int maximum = array[0];
  for (int i = 1; i < length; i++) {
    if (maximum < array[i]) {
      maximum = array[i];
    }
  }
  return maximum;
}

int main() {
  int array[] = {2, 78, -4, 5, 19, 0};
  max(array, 6);
}
